topic-operator/src/test/java/io/strimzi/operator/topic/LogCaptor.java

```diff
@@ -72,7 +72,7 @@ private LogCaptor(AbstractLogger logger,
      * @param predicate A predicate (e.g. on the message text)
      * @param timeout A timeout
      * @param unit A timeout unit
-     * @return
+     * @return created LogCaptor
```

Comment:
You should start with a capital letter. Same below as well ...

```suggestion
     * @return Created LogCaptor
```